_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -552,12 +554,29 @@ resource "aws_ebs_encryption_by_default" "ebs_encryption" {
 }
 ----
 
+[[configure_rds_encryption]]
+==== Configure RDS Encryption
+To comply with recommendation 2.3.1, if you use RDS it must be configured with link:http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html[RDS encryption] for all instances and in all of the enabled AWS regions within your AWS Account(s). You can set to `true` the Terraform `storage_encrypted` attribute for an RDS instance to implement the recommendation.
+
+For example:
+
+[source,hcl]
+----
+resource "aws_db_instance" "your_primary_db" {
+  storage_encrypted = true
+}
+----
+
+[[configure_s3_data]]
+==== Configure your S3 Data
```

Comment:
```suggestion
==== Secure/Protect/? your S3 Data
```

Not sure which word fits better. Enabling MFA delete is to help protect the data and Macie is to monitor the data for Personal Identifiable Information

Reply:
+1, "Secure your S3 data" sounds good to me.

Reply:
That's great context, Marina! Will change it to "Protect" - I think it fits better.

Reply:
oh shit, only just seen this! I'm happy to put it back to Secure.... if both of you think it's "secure" that's the one. I thought "protect" fits better here since securing is more for encryption and so on, whilst "protect" is to limit access, or organise it in a .... secure 😁 way